contracts/Market.sol

```diff
@@ -799,21 +788,20 @@ contract Market is
     bytes32 r,
     bytes32 s
   ) external whenNotPaused {
-    uint256 certificateAmount = this
-      .calculateCertificateAmountFromPurchaseTotal({purchaseTotal: amount});
```

Comment:
Previously, the swap endpoints received the total amount of currency needed for the purchase, in units of the purchasing token, and would back out the certificate amount from this value based on the price multiple and the marketplace fee.  Now, for simplicity and to avoid errors/misunderstandings on the part of the buyer, the `amount` value passed to any purchasing endpoint is the number of NRTs to buy, and all other values/amounts owed to the various parties are calculated from that.